ej2-react/grid/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
 title: Getting started with React Grid component | Syncfusion
-description:  Checkout and learn about Getting started with React Grid component of Syncfusion Essential JS 2 and more details.
+description: Checkout and learn about Getting started with React Grid component of Syncfusion Essential JS 2 and more details.
 control: Getting started 
 platform: ej2-react
 documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Getting started
+# Getting started with React Grid component
```

Comment:
In H1 remove the word component. Check in all places

Reply:
Done